mailtrap/models/contacts.py

```diff
@@ -95,3 +95,34 @@ class Contact:
 @dataclass
 class ContactResponse:
     data: Contact
+
+
+class ContactImportStatus(str, Enum):
```

Comment:
```suggestion
class ContactImportStatus(StrEnum):
```
There is a special class in enum package

Reply:
StrEnum is released in python3.11, we support 3.9+.
It's possible with third part libraries, but I'm not sure it's a good idea to install something else just for StrEnum

Reply:
I didn't check it, then it's fine, thanks